### ISSUE TITLE
Technical metadata requirements

### ISSUE THREAD

**Issue by arwenhutt** [ucsdlib/horton#50]
### Descriptive summary
Vet standard cc/hyrax technical metadata properties with Stefan and Annalise to identify anything missing and which properties they may be able to provide as output from their processing.  This would be a good time to also look at the alignment between the DAMS4 terms and the DAMS5 terms and create an initial mapping.

### Deliverables
1) List of any technical metadata properties proposed to be added with use cases
2) List of any technical metadata fields to be included in the Excel ingest spreadsheet
3) Mapping of current D4 technical metadata fields to D5

### Related work
https://github.com/ucsdlib/dams5-cc-pilot/issues/23

**Comment by remerjohnson** [comment]
The [technical metadata exports from the cc pilot](https://github.com/ucsdlib/dams5-cc-pilot/issues/23#issuecomment-255160183) are relevant so wanted to call those out here. 

Also, a more high-level view is available as the [Technical Metadata Application Profile](https://wiki.duraspace.org/display/hydra/Technical+Metadata+Application+Profile)

**Comment by remerjohnson** [comment]
Still no deliverables yet, but we had the [preservation meeting](https://lisn.ucsd.edu/display/MSCL/2017-03-22+Born-digital+storage+and+preservation) yesterday and got a better sense of the types of born-digital materials they were expecting on having in the near future.  

There was a heavy emphasis on disk images, including entire hard drive images, floppy, and images taken from other physical media. So one thing I might do is open a ticket to get an export of the technical metadata from ingesting an .iso into Horton. That way I can come to Annalise and Stefan with a base of metadata they can evaluate. 

We also veered into modeling, which was tricky but they grasped the rather hack-y way we currently support up to 2 "File Use" values and how that could support some of what they are thinking... e.g. zipping up a FITS and/or BitCurator export and having that as a `data-source` as the second File Use for an object. Alternatives would be having them as separate (curator-only) objects.

**Comment by mcritchlow** [comment]
@remerjohnson any updates or changes here? Based on the last part of your last comment, this might be relevant for #111 ?

**Comment by remerjohnson** [comment]
@mcritchlow It might be. I'll probably talk about it in #111 but it would probably be helpful if we could have more than 2 File Use values, because of certain use cases coming in for transcripts, FITS exports, OCR, etc. 

The other option, but I'm not sure about implementation, would be to have something like `hasRelatedFile` for the FITS or other XML output as recommended in the [Technical Metadata Application Profile](https://wiki.duraspace.org/display/hydra/Technical+Metadata+Application+Profile#TechnicalMetadataApplicationProfile-DetailedTechnicalMetadataCapturedasXML). If there's strong thoughts about that, let me know.

**Comment by mcritchlow** [comment]
@remerjohnson - ok. I'm just wondering if we need to keep this open or not. It's been hanging for a few months, so thought I'd check in. Thanks

**Comment by remerjohnson** [comment]
@mcritchlow Yeah good point. I'm reaching out to Annalise (and Stefan) again to nail this down and resolve lingering stuff like disk images. Then if there's anything special we need I'll update here (or just close this out if nothing else needed).

**Comment by remerjohnson** [comment]
@mcritchlow So my discussions with Annalise have now turned to UI considerations for things like disk images, and not technical metadata. I think we can move forward with what we have. Also, Samvera Virtual Connect yesterday helped clear up some concerns about these types of objects. I don't have any outstanding concerns.
